fmriprep.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import sys
+import os
+import argparse
+import subprocess
+
+code_base = '/data/chris/Projects/poldracklab'
+data_base = '/data/fmriprep/data'
+working = '/data/fmriprep/working'
+
+
+def main(cmd, *argv):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-f', '--patch_fmriprep', action='store_true')
+    parser.add_argument('-w', '--patch_niworkflows', action='store_true')
+    parser.add_argument('-n', '--patch_nipype', action='store_true')
+    parser.add_argument('--fmriprep_path', type=str,
+                        default=os.path.join(code_base, 'fmriprep'))
```

Comment:
I would avoid defaults here since they are specific to a particular machine (in this case your are setting those folders to `/data/chris/...`